Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3276,14 +3283,24 @@ def handle_eml(file_path, b64=False, file_name=None):
         headers = parser.parsestr(file_data)
 
         header_list = []
+        headers_map = {}
         for item in headers.items():
             item_dict = {
                 "name": item[0],
                 "value": convert_to_unicode(item[1])
             }
 
+            # old way to map headers
             header_list.append(item_dict)
 
+            # new way to map headers - dictionary
+            headers_map[item[0]] = convert_to_unicode(item[1])
```

Comment:
Need to think this through. We may be overriding an existing header when an email contains multiple headers of the same name.

Reply:
fixed